sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/rx/examples/multimaster/samples/ConflictWorker.java

```diff
@@ -530,9 +528,7 @@ private boolean hasDocumentClientExceptionCause(Throwable e, int statusCode) {
         BridgeInternal.setProperty(document, "regionEndpoint", client.getReadEndpoint());
 
         RequestOptions options = new RequestOptions();
-        options.setAccessCondition(new AccessCondition());
-        options.getAccessCondition().setType(AccessConditionType.IF_MATCH);
-        options.getAccessCondition().setCondition(document.getETag());
+        options.setIfMatchEtag(document.getETag());
```

Comment:
This is called ETag elsewhere (see `CosmosAsyncItemResponse.java:    public String getETag()`). you Have it as Etag. we should be consistent on this.

Reply:
please Fix this in your PR before merging. other than that LGTM.

Reply:
Consistency is key.

Reply:
Cosmos have logic ifMatchEtag, and ifNoneMatchEtag. because of that we cant put setEtag on options